Tella/Data/Database/TellaDataBase.swift

```diff
@@ -73,7 +73,8 @@ class TellaDataBase: UwaziServerLanguageProtocol {
             cddl(D.cApiProjectId, D.text),
             cddl(D.cSlug, D.text),
             cddl(D.cAutoUpload, D.integer),
-            cddl(D.cAutoDelete, D.integer) ]
+            cddl(D.cAutoDelete, D.integer),
+            cddl(D.cServerType, D.text)]
```

Comment:
Is this the right way to add a column into a new table? I believe I should create a migration first to alter the current table but I didn't see any other migration in the codebase. 

cc @RobertShrestha